### DISCUSSION TITLE
Please Migrate away from Discord

### DISCUSSION THREAD

**Discussion by krumelmonster** [blue-build/.github#19] [General]
Dear blue-build community,

I've long been looking for a project like blue-build and I'd love to catch up in a community chat but discord won't let me join the conversation without providing a mobile number and that is after I've already provided a nickname, my birth date, my mail address and solved two captchas.

I don't feel comfortable providing this amount of unrelated information to a commercial and centralized platform and I feel "tricked" into providing them as it initially seemed I could join the conversation right after providing the nickname.

I'd like to suggest migrating to a matrix based chat which can be accessed with a webinterface similar to Discord yet is based on an open protocol with various clients for various platforms and needs, is federated, bridgeable, open source and also widespread enough in the community that many people may have an account already.

<details><summary>Discord asking for phone number</summary>
<p>

![image](https://github.com/blue-build/.github/assets/13542760/ab2c8c5b-bd64-4092-b961-ef3814eb8a26)

</p>
</details>

**Comment by xynydev** [comment]
Something related was asked on Discord just yesterday (by @Henry-Hiles)! Here's and another community member's (@Janybanny) response:
![image](https://github.com/blue-build/.github/assets/60004820/158f697e-3045-4235-b07b-e175aef3e4ac)
![image](https://github.com/blue-build/.github/assets/60004820/c69072b2-93d0-4a25-8e86-256fa6a91658)

As such, I think it's unlike we'll completely migrate away from Discord due to the network effect. Hosting a bridge is something I am not currently considering doing myself, but if a community member wants to step up to do it, and that can be done in a secure way, I wouldn't be against it.

And about Matrix specifically, while I subscribe to the ideals of federation etc., I find the UX somewhat lacking, and the moderation and community management tools too. https://revolt.chat/ delivers better on the UX front IMO, it would even fit as the only place for a community to exist.

---

That being set, the Discord guild is not set to require a phone number, just for your account to be older than five minutes. It seems like you are creating a brand-new account. Could you try logging in again to see if it still asks for your phone number?

![image](https://github.com/blue-build/.github/assets/60004820/16289656-52c5-4c29-92b3-b85f020d6c65)

If you're paranoid about using Discord, but still want to give it a shot, I'd recommend you use it in your web browser with an adblocker only. It might also help to provide a real email address (it doesn't need to be your main one, but not an alias or tempmail either), to make it less likely that it ask you to provide a phone number. I'm not the biggest fan of Discord in the privacy and security aspect either, and nowadays the mobile UI sucks too, but the network effect is a bitch. I don't think I even seriously considered another platform to host the chat.

Furthermore, I'm trying to make the community not depend on Discord through the use of this Discussion forum. You can hang out here, and I'll try to make sure your questions are always answered. It's a shame that Discord (and synchronous forms of communication) usually are more active. It really would be nice if more people used the forum.

**Reply by Henry-Hiles** [reply]
We don't need to self-host a bridge, you can use a public instance like https://t2bot.io, only downside is a small (but manageable) amount of lag. I'd be open for setting up the matrix space for you if you want!

**Reply by krumelmonster** [reply]
I would greatly appreciate that

**Reply by xynydev** [reply]
We'll probably be trying to set that up today.

**Reply by xynydev** [reply]
@krumelmonster We just set up a bridged Matrix space. [#bluebuild:matrix.org](https://matrix.to/#/#bluebuild:matrix.org)

You can avoid using a Discord account with this, but unfortunately your messages will probably still be stored and analyzed by Discord...

**Reply by krumelmonster** [reply]
Wonderful, thank you so much!
